app/serializers/server_room_serializer.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 
 class ServerRoomSerializer < RoomSerializer
-  attributes :owner, :status, :participants
+  attributes :owner, :status, :participants, :last_session
 
   def owner
     object.user.name
   end
+
+  def last_session
+    return 'No meeting yet' if object.last_session.nil?
+
+    last_session_date = object.last_session.strftime('%B %e, %Y')
+    object.status == 'Active' ? "Started: #{last_session_date}" : "Ended: #{last_session_date}"
```

Comment:
I know that `status` is out of the scope of this PR.
But, just to mention a question that came to my mind; Is the status non binary? 
Is there a status other than `Active|Not Running`?
If not I think it may make more sense to switch the `status` to `active` Boolean attribute and leave this responsibility for the frontend to map it to an informational output such us `Active`.

Reply:
What do you think about this @farhatahmad ? 
Could there eventually be a room status that is neither Active or Not Running ?
Should we make a switch from status:string to active:boolean ?